src/f3dasm/_src/experimentdata/_io.py

```diff
@@ -0,0 +1,184 @@
+"""
+Module to load and save output data of experiments \
+and other common IO operations
+"""
+
+#                                                                       Modules
+# =============================================================================
+
+from __future__ import annotations
+
+# Standard
+import pickle
+from pathlib import Path
+from typing import Any, Mapping, Optional, Type
+
+# Third-party
+import numpy as np
+import pandas as pd
+import xarray as xr
+
+# Local
+from ..logger import logger
+
+#                                                          Authorship & Credits
+# =============================================================================
+__author__ = 'Martin van der Schelling (M.P.vanderSchelling@tudelft.nl)'
+__credits__ = ['Martin van der Schelling']
+__status__ = 'Stable'
+# =============================================================================
+
+#                                                  Global folder and file names
+# =============================================================================
+
+EXPERIMENTDATA_SUBFOLDER = "experiment_data"
+
+LOCK_FILENAME = "lock"
+DOMAIN_FILENAME = "domain"
+INPUT_DATA_FILENAME = "input"
+OUTPUT_DATA_FILENAME = "output"
+JOBS_FILENAME = "jobs"
+
+#                                                               Storing methods
+# =============================================================================
+
+class _Store:
+    suffix: int
+
```

Comment:
There are a lot of docstrings missing in this new module. Please add them, even if the methods are simple and/or private.